tests/foreman/installer/test_infoblox.py

```diff
@@ -4,6 +4,8 @@
 
 :CaseLevel: System
 
+:CaseAutomation: Automated
+
 :CaseComponent: DHCPDNS
```

Comment:
How about adding it to a specific component?
```suggestion
:CaseComponent: InfobloxIntegration
```

Reply:
Updated